dangerfile.js

```diff
@@ -15,18 +15,20 @@ const clickupRequest = async ({ resource, method = 'POST', data = {} }) =>
       'Content-Type': 'application/json',
     },
     data,
-  }).catch(console.log);
+  }).catch(
+    function (error) {
+      console.log(`Error getting ${resource}`)
+    }
+  );
 
 const getClickupTicketName = async (taskId, isCustom) => {
   const resource = getTaskResource({ taskId, isCustom }, "")
 
-  const {
-    data: { name },
-  } = await clickupRequest({
+  const response = await clickupRequest({
     resource,
     method: 'GET',
   });
-  return name
+  return response?.data?.name ?? undefined
```

Comment:
```suggestion
  return response?.data?.name
```

nit: `??` is redundant. `?.` optional chaining will already return `undefined` if any property is missing.